tests/time_tests/test_runner/conftest.py

```diff
@@ -195,6 +212,45 @@ def validate_test_case(request, test_info):
     yield
 
 
+@pytest.fixture(scope="function")
+def parse_and_validate_db_info(request, test_info):
+    """Fixture for parsing and validating metadata to submit to a database.
+
+    Fixture prepares metadata to submit to a database. One of the steps is parsing of
+    build information from build manifest. After preparation, it checks if metadata
+    contains required properties.
+    """
+    run_id = request.config.getoption("db_submit")
+    if not run_id:
+        yield
+        return
+
+    if request.config.getoption("metadata"):
+        test_info["db_info"].update(json.loads(request.config.getoption("metadata")))
+    manifest_metadata = metadata_from_manifest(request.config.getoption("manifest"))
+    test_info["db_info"].update(manifest_metadata)
+
+    schema = {
+        "type": "object",
+        "properties": {
+            "platform": {"enum": ["linux_ubuntu_18_04", "windows_vs2019"]},
+            "repo_url": {"type": "string"},
+            "commit_sha": {"type": "string"},
+            "commit_date": {"type": "string"},
+            "target_branch": {"type": "string"},
+            "ovino_version": {"type": "string"}
+        },
+        "required": ["platform", "repo_url", "commit_sha", "commit_date", "target_branch", "ovino_version"]
+    }
```

Comment:
I think we need to make schema a string and parse it on startup.

Reply:
Done